src/middle/Backend_intf.ml

```diff
@@ -0,0 +1,14 @@
+open Mir
+
+module type S = sig
```

Comment:
Is this at all related to the PR?

Reply:
No - I tidied it up before I realized I wasn't going to be able to use the signatures yet.